Warning on XCode 11

Aug 18 master:

```
/Users/dellaert/git/SwiftFusion/Tests/SwiftFusionTests/Inference/ValuesStorageTests.swift:83:20: warning: parameter 'file' with default argument '#filePath' passed to parameter 'file', whose default argument is '#file'
        """, file: file, line: line)
                   ^
/Users/dellaert/git/SwiftFusion/Tests/SwiftFusionTests/Inference/ValuesStorageTests.swift:75:26: note: did you mean for parameter 'file' to default to '#file'?
    file: StaticString = #filePath,
                         ^~~~~~~~~
                         #file
/Users/dellaert/git/SwiftFusion/Tests/SwiftFusionTests/Inference/ValuesStorageTests.swift:83:20: note: add parentheses to silence this warning
        """, file: file, line: line)
                   ^
                   (   )
<unknown>:0: note: 'file' declared here
/Users/dellaert/git/SwiftFusion/Tests/SwiftFusionTests/Inference/ValuesStorageTests.swift:86:63: warning: parameter 'file' with default argument '#filePath' passed to parameter 'file', whose default argument is '#file'
    XCTAssertEqual(Array(typedActual), Array(expected), file: file, line: line)
                                                              ^
/Users/dellaert/git/SwiftFusion/Tests/SwiftFusionTests/Inference/ValuesStorageTests.swift:75:26: note: did you mean for parameter 'file' to default to '#file'?
    file: StaticString = #filePath,
                         ^~~~~~~~~
                         #file
/Users/dellaert/git/SwiftFusion/Tests/SwiftFusionTests/Inference/ValuesStorageTests.swift:86:63: note: add parentheses to silence this warning
    XCTAssertEqual(Array(typedActual), Array(expected), file: file, line: line)
                                                              ^
                                                              (   )
<unknown>:0: note: 'file' declared here

```

--- a/Tests/BeeDatasetTests/BeeDatasetTests.swift
+++ b/Tests/BeeDatasetTests/BeeDatasetTests.swift
@@ -48,7 +48,7 @@ final class BeeDatasetTests: XCTestCase {
 
 extension URL {
   /// Creates a URL for the directory containing the caller's source file.
-  static func sourceFileDirectory(file: String = #file) -> URL {
+  static func sourceFileDirectory(file: String = #filePath) -> URL {
     return URL(fileURLWithPath: file).deletingLastPathComponent()
   }
 }

--- a/Tests/SwiftFusionTests/TestUtilities.swift
+++ b/Tests/SwiftFusionTests/TestUtilities.swift
@@ -98,7 +98,7 @@ public enum SimpleGaussianFactorGraph {
 
 extension URL {
   /// Creates a URL for the directory containing the caller's source file.
-  static func sourceFileDirectory(file: String = #file) -> URL {
+  static func sourceFileDirectory(file: String = #filePath) -> URL {
     return URL(fileURLWithPath: file).deletingLastPathComponent()
   }
 }

--- a/Tests/SwiftFusionTests/TestUtilities.swift
+++ b/Tests/SwiftFusionTests/TestUtilities.swift
@@ -11,7 +11,7 @@ func assertEqual<T: TensorFlowFloatingPoint>(
   guard x.shape == y.shape else {
     XCTFail(
       "shape mismatch: \(x.shape) is not equal to \(y.shape)",
-      file: file,
+      file: (file),
       line: line
     )
     return
@@ -19,7 +19,7 @@ func assertEqual<T: TensorFlowFloatingPoint>(
   XCTAssert(
     abs(x - y).max().scalarized() < accuracy,
     "value mismatch:\n\(x)\nis not equal to\n\(y)\nwith accuracy \(accuracy)",
-    file: file,
+    file: (file),
     line: line
   )
 }
@@ -39,7 +39,7 @@ func assertAllKeyPathEqual<T: KeyPathIterable>(
     XCTAssert(
       false,
       "value mismatch:\n\(x)\nis not equal to\n\(y)\nwith accuracy \(accuracy)",
-      file: file,
+      file: (file),
       line: line
     )
   }

